### PR TITLE
Chore: Add Happy Hues link to colors subcategory(Typo error fixed)

### LINKED ISSUE #733: [Add]Happy Hues to colors subcategory
### Site URL

https://www.happyhues.co/

### Description of the above provided site

Not sure what colors to use in your designs or where to use them? Happy Hues is a color palette inspiration site that acts as a real world example as to how the colors could be used in your design projects.

### Category

Frontend

### Sub Category

colors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Oyebolade**
## Fixes Issue
Closes #733

## Changes proposed
Happy Hues link was added to colors subcategory.

## Screenshots
Happy Hues

### CODE REVIEW

File: database/frontend/colors.json [hunk 1]
```diff
@@ -89,5 +89,12 @@
       "url": "https://www.colorion.co/",
     "category": "frontend",
     "subcategory": "colors"
+  },
+   {
+    "name": "Happy Hues",
+    "description": "Not sure what colors to use in your designs or where to use them? Happy Hues is a color palette inspiration site that acts as a real world example as to how the colors could be used in your design projects.",
```
Comment (CBID2) [inline database/frontend/colors.json:95]: [Proofreading]
```suggestion
    "description": "Not sure what colors to use in your designs or where to use them? Happy Hues is a color palette inspiration site that acts as a real-world example of how the colors could be used in your design projects.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Oyebolade! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

File: database/frontend/colors.json (hunk 1 above)
Comment (Oyebolade) [reply database/frontend/colors.json:95]: All right, I will fix it.
Thank you

**Comment by Oyebolade** [comment]
> Hi @Oyebolade! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

The errors has been fixed.
Thanks.

File: database/frontend/colors.json [hunk 2]
```diff
@@ -96,5 +96,12 @@
       "url": "https://www.colorion.co/",
     "category": "frontend",
     "subcategory": "colors"
+  },
+   {
+    "name": "Happy Hues",
+    "description": "Not sure what colors to use in your designs or where to use them? Happy Hues is a color palette inspiration site that acts as a real-world example of how the colors could be used in your design projects.",
```
Comment (rupali-codes) [inline database/frontend/colors.json:102]:
```suggestion
    "description": "Not sure what colors to use in your designs or where to use them? Happy Hues is a color palette inspiration site that acts as a real-world example of how the colors could be used in your design projects.",
```
```suggestion
    "description": "Not sure what colors to use in your designs or where to use them? Happy Hues is a color palette inspiration site that acts as a real-world example of how the colors could be used in your projects.",
```

Comment (Oyebolade) [reply database/frontend/colors.json:102]: All right

Comment (Oyebolade) [reply database/frontend/colors.json]: Corrected

**Review by rupali-codes — APPROVED** [review]
looks good

**Comment by rupali-codes** [comment]
@CBID2 ☺️

**Pull request merged**